src/hooks/useSignInRedirect.ts

```diff
@@ -29,9 +31,17 @@ export function useSignInRedirect() {
   const requestAuthentication = useCallback(
     (createAccount = false) => {
       saveCurrentUrl();
-      router.push(createAccount ? '/signup' : '/signin');
+      if (!vmNear) return;
+      vmNear.selector
+        .then((selector: any) => selector.wallet('fast-auth-wallet'))
+        .then((fastAuthWallet: any) =>
+          fastAuthWallet.signIn({
+            contractId: vmNear.config.contractName,
+            isRecovery: !createAccount,
+          }),
+        );
```

Comment:
These changes completely break our `Sign In` and `Create Account` buttons in the build preview. We can't remove this logic:

```
router.push(createAccount ? '/signup' : '/signin');
```

Without directing the user to these pages, they won't be able to choose if they want to sign in with FastAuth or with their wallet. We can't assume users will only sign in with FastAuth.

@charleslavon do you have any context on these changes?